cmd/kube-proxy/app/server.go

```diff
@@ -127,16 +130,24 @@ func (s *ProxyServer) Run(_ []string) error {
 		glog.V(2).Infof("Running in resource-only container %q", s.ResourceContainer)
 	}
 
+	if s.Master != "" {
+		if len(s.APIServerList) != 0 {
+			glog.Warningf("Both --mater and --api-servers set. Using --api-servers")
```

Comment:
@jayunit100 is suggesting we use `glog.Fatalf()` here (I asked him offline). I don't think I like that. While having both is clearly wrong, we can still function and I feel like we should continue to try to function. It seems most places we try to be as resilient as possible to errors. I can certainly make it hard fail if anyone else things it should be a hard failure.